Sources/PassKit/FakeSendable.swift

```diff
@@ -0,0 +1,8 @@
+// This is a temporary fix until RoutesBuilder and EmptyPayload are not Sendable
```

Comment:
You're better off using `@preconcurrency import` which will provide better sendable checking rather than blindly marking it as unchecked Sendable which is unsafe